terraform/modules/psn/data/psn-network-policy.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: crd.projectcalico.org/v1
+kind: GlobalNetworkPolicy
+metadata:
+  name: psn-egress-policy
+spec:
+  order: 90
+  types:
+  - Egress
+  egress:
+  - action: Deny
+    protocol: TCP
+    source:
+      selector: talksToPsn != 'true'
```

Comment:
alright so we'll need to communicate the requirement to the DCS team that the prod HMPO agent will need this annotation

Reply:
Yeah. GSP changes will need to go in first, before any of this will take effect. I'm also not sure everyone will agree the `kubectl apply` approach I've added into this PR is The Right Way:tm:. That's if it works at all :smile: